src/storage/memory.rs

```diff
@@ -77,6 +77,11 @@ impl MemoryStorage {
         self.memory = self.transacted.clone();
         self.persisted = self.transacted.clone();
     }
+
+    /// Set the block height of the chain
+    pub fn set_block_height(&mut self, block_height: u32) {
```

Comment:
Ditto; this is responsibility of the storage and shouldn't be exposed to the user via VM.

Reply:
This is on the in-memory storage only used in tests though, users aren't expected to use this. I wanted the ability to preserve storage between transaction - which MemoryStorage via commit and rollback, while modifying the block height during tests.

Reply:
used `test-helpers` as that seems like the best flag available in the VM.